src/api/Alert/types.ts

```diff
@@ -135,7 +135,8 @@ export interface IGetCaseParams extends IGetSubGraphParams {
 export enum CaseStatus {
   UNCONFIRMED = 'unconfirmed',
   CONFIRMED = 'confirmed',
-  DISMISSED = 'dismissed'
+  DISMISSED = 'dismissed',
+  IN_PROGRESS = 'in_progress'
```

Comment:
Let's use "in-progress" for consistency.
e.g: SUPER_SAMPLING, USE_CREDENTIALS, NON_GRAPH, INVALID_EXPRESSION_SYNTAX ...